src/components/listitems/ListItemTransactionRadio.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { View } from "react-native";
+import RNReactNativeHapticFeedback from "react-native-haptic-feedback";
+import { IOStyles, useIOExperimentalDesign } from "src/core";
+import { Icon } from "../icons";
+import { AnimatedRadio } from "../radio/AnimatedRadio";
+import { HSpacer, VSpacer } from "../spacer";
+import { H6 } from "../typography";
+import { LabelSmall } from "../typography/LabelSmall";
+import { PressableListItemBase } from "./PressableListItemsBase";
+
+type ListItemTransactionRadioProps = {
```

Comment:
We could try to find a new name, considering that this is a specific instance of a `ListItemRadio` with a clear purpose. We already know that `amountString` is mandatory, so a more specific name could be added. For example, `ListItemRadioChooseAmount` or something similar.

Reply:
To be honest, `amountString` to me sounds like the correct name since it **is** an amount -- just not a straight amount, because we don't do any formatting, and it sounds specific enough to me.
Maybe `formattedAmountString` since it is an amount string, which has to be formatted prior to using it; what do you think?

Reply:
Mine was not a comment on the prop name, which is perfectly fine (I like your proposal, too). I was referring to the name of the entire component. Maybe I should have pointed the comment to the correct line.

Reply:
oh welp, my bad 😅 
Since it is marked as `ListItemTransaction` in figma, I thought it would make sense to name it like that, especially since it's strongly bound to transactions; also I think "chooseAmount" doesn't really do it justice, since there is no "multiple choice" in the component itself; what do you think?

Reply:
> also I think "chooseAmount" doesn't really do it justice, since there is no "multiple choice" in the component itself; what do you think?

It's a good observation, but a single `RadioItem` doesn't make sense. If there are at least two `RadioItem` components, with `amountString` as a mandatory prop, we have a `ChooseAmount` scenario. That's why I suggested that name.

My main concern is that `ListItemTransactionRadio` doesn't explain its purpose very well. How about `ListItemRadioTransactionWithAmount` or `ListItemRadioWithAmount`? I know it's pretty wordy, but better to be clear than cryptic.

(It is better to use `ListItemRadio...` than `ListItemTransaction...`, because with the former we have automatically grouped the different `ListItemRadio` components)

Reply:
addressed in [31959ac](https://github.com/pagopa/io-app-design-system/pull/139/commits/31959aca9ea86cd855145e281a7776ed49167f30), LMK what you think about the `ListItemRadioWithAmount` name 👍